src/init/__tests__/add-contributors-list.js

```diff
@@ -1,4 +1,6 @@
+import {unlink} from 'fs'
```

Comment:
[minor/non blocking] would be good if we could mock the file system using something like https://www.npmjs.com/package/mock-fs

Reply:
I've tried `mock-fs` which works fine for testing files that _already exist_ but for those who don't (i.e. `LOREM.md`) which are meant to be deleted as soon as the test suite finishes, it's not doable as the code uses `fs` to write to files (when it doesn't exist) and `mock-fs` doesn't have any `unlink` like method.

Mind you, I could get around that by using `fs.unlink` but I'm not sure if it would be worthwhile with `mock-fs`.

Reply:
@tbenning What do you think?

Reply:
@jakebolam Shell I go ahead and merge this or look into another option to mock the FS?